### ISSUE TITLE
[Crash]: Crashing upon ticking world

### ISSUE THREAD

**Issue by Engrene4** [AllTheMods/ATM-10#1026]
### Possible Fixes

Yes

### Modpack Version

1.15

### What happened?

I started a world on the 1.11 version and updated the modpack as updates rolled out. When on version 1.15 is when issues started, first crashing upon loading into other dimensions, next crashing upon just walking around and loading chunks. Tried troubleshooting by removing the ServerCore mod added in 1.15 and increasing memory. It proceeded to work for another day until now it is happening again the next day after re-launching the modpack. (I am hosting a LAN server)

### Crash Log

https://gist.github.com/Engrene4/62eebe08d8630a8b3f94b97bb71654c8

### Latest.Log

https://gist.github.com/Engrene4/29bc084b33e784bf1623db4afccf8b0a

**Comment by Engrene4** [comment]
I might have figured it out, I had a chunk force loaded 50K blocks away and a chunk loaded in another dimension I removed them and am currently fine. I will see how long it lasts.

**Comment by Engrene4** [comment]
The world is gone, I went into the other and crashed and there was a strange particle being unable to render. when I removed some mods due to more crashing because of sodium and iris I just reinstalled everything to see if I could get out of the other in single player, but when that did not work I decided to backup the world only to find my backups deleted because of the reinstallation through curse forge (the "reinstallation" was actually clicking the same version of the modpack and telling it to update to its same version because repair profile did not work) all of the backups were gone and I still crashed every time I got in. I just started a new world and am never updating it.
